aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,315 @@
+Task Name	   Description    
+=============  ===============================================================================================================================  
+clean:js	   A task that uses the rimraf Node deletion module to remove unneeded text from *.js* files.
+clean:css	   A task that uses the rimraf Node deletion module to remove unneeded text from *.css* files.
+clean	       A task that calls both the ``clean:js`` and ``clean:css`` tasks.
+min:js	       A task that matches files based on the specified JavaScript file path, concatenates the files into a destination location, removes unneeded characters using the ``uglify()`` function, and re-writes the file to the destination location.
+min:css	       A task that matches files based on the specified CSS file path, concatenates the files into a destination location, minimizes the CSS file using the ``cssmin()`` function, and re-writes the file to the destination location.
+min	           A task that calls both the ``min:js`` and ``min:css`` tasks.
+=============  =============================================================================================================================== 
+
+Running Default Tasks
+---------------------
+
+If you haven’t already created a new Web app, create a new ASP.NET Web Application project in Visual Studio 2015.
+
+1.	Select **File** > **New** > **Project** from the menu bar. The **New Project** dialog box is displayed.
+
+	.. image:: bundling-and-minification/_static/01-NewProjectDB.png
+	
+2.	Select the **ASP.NET Web Application** template, choose a project name and click **OK**.
+3.	In the **New ASP.NET Project** dialog box select the **Web Application** template from the **ASP.NET 5 Templates** and click **OK**.
+4.	In **Solution Explorer**, right-click *gulpfile.js* and select **Task Runner Explorer**. 
+
+	.. image:: bundling-and-minification/_static/02-SolutionExplorer-TaskRunnerExplorer.png
+	
+	**Task Runner Explorer** shows the list of gulp tasks. In the default ASP.NET 5 Web Application template in Visual Studio 2015 there are six tasks included from *gulpfile.js*.
+
+	.. image:: bundling-and-minification/_static/03-TaskRunnerExplorer.png 
+
+5.	Underneath **Tasks** in **Task Runner Explorer** right-click **clean** and select **Run** from the pop-up menu.
+
+	.. image:: bundling-and-minification/_static/04-TaskRunner-clean.png 
+
+**Task Runner Explorer** will create a new tab named **clean** and execute the related clean task as it is defined in *gulpfile.js*.
+
+6.	Next, right-click the **clean** task, then select **Bindings** > **Before Build**.
+
+ 	.. image:: bundling-and-minification/_static/05-TaskRunner-BeforeBuild.png 
+
+	The **Before Build** binding option will allow the clean task to be automatically run before each time you build your project.
+
+Defining and Running a New Task
```

Comment:
This section is not really about bundling or minification - belongs in the "Using Gulp" topic